src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -189,6 +189,11 @@
     short-summary: get publishing endpoints, credentials, database connection strings, etc
 """
 
+helps['webapp deployment show-container-cd-url'] = """
```

Comment:
Based on the [discussion](https://github.com/Azure/azure-cli/issues/4023#issuecomment-316490764), the command should be under `webapp deployment container`. Please correct it. Your change in the commands file is right